chapters/classes.tex

```diff
@@ -387,7 +387,7 @@ \subsection{Component Variability Prefixes discrete, parameter, constant}\double
   constant during transient analysis.
 \item
   A \emph{discrete-time} variable vd has a vanishing time derivative
-  (informally \lstinline!der(vd)=0! between events and possibly infinite derivative at event,
```

Comment:
Ah, you've removed the part I previously thought should be reworded. Fine by me.